src/main/java/at/hannibal2/skyhanni/features/garden/contest/FarmingPersonalBestGain.kt

```diff
@@ -14,8 +17,9 @@ import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 @SkyHanniModule
 object FarmingPersonalBestGain {
-    private val config get() = GardenAPI.config
+    private val config get() = GardenAPI.config.personalBests
     private val patternGroup = RepoPattern.group("garden.contest.personal.best")
+    private var pbIncrement = mapOf<CropType, Int>()
```

Comment:
```suggestion
    private var personalBestIncrements = mapOf<CropType, Int>()
```

This should also be pluralised and the name doesnt need to be shortened as this will just lead to confusion later on